docs/data/README.mdx

```diff
@@ -2,10 +2,28 @@
 title: Data Overview
 ---
 
-The RPC, data indexers, Hubble, and Horizon are platforms or tools that facilitate communication between users and applications and the Stellar network. With these various platforms, users can query network data, submit transactions, and interact with smart contracts without needing to understand the low-level details of Stellar Core.
+The RPC, data indexers, analytical platforms, Galexie, Hubble, and Horizon are platforms or tools that facilitate communication between users and applications and the Stellar network. With these various platforms, users can query network data, submit transactions, and interact with smart contracts without needing to understand the low-level details of Stellar Core.
 
 This section will walk you through the differences between the various platforms and tools, what platform or tool is best for what use case, and then link to their various documentation locations.
 
+| Features               | RPC | Horizon | Galexie | Hubble |
+| ---------------------- | --- | ------- | ------- | ------ |
+| Real-time Data         | ✅  | ✅      | ✅      | ❌     |
+| Historical Data        | ❌  | ❌      | ✅      | ✅     |
+| Classic Data           | ✅  | ✅      | ✅      | ✅     |
+| Soroban Data           | ✅  | ❌      | ✅      | ✅     |
```

Comment:
nit: update to "Smart Contracts"

Reply:
updated in updated in a088c83120488cfcf7ef46830ab6621107828e2c